src/main/proto/cosmas.proto

```diff
@@ -0,0 +1,16 @@
+message Text {
+  string text = 0

```

Comment:
[From the docs](https://developers.google.com/protocol-buffers/docs/proto3#assigning-tags):
> The smallest tag number you can specify is 1